src/resolvers/__tests__/user.resolvers.spec.ts

```diff
@@ -51,4 +53,46 @@ describe('Test User Resolver', () => {
       expect(error.message).toEqual(message);
     }
   });
+
+  // password reset
+  it('Should send an email to the user if exists', async () => {
+    const email = 'foobar@gmail.com';
+    const spy = jest.spyOn(userResolver.Mutation, 'sendResetPasswordEmail');
+    await userResolver.Mutation.sendResetPasswordEmail(null, { email });
```

Comment:
@harerakalex Could we mock the sending of emails while testing. Because we shall be running tests so many times while developing our application. I think tests will always fail once your internet is off.